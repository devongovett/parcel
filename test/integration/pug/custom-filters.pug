doctype html
html(lang="en")
  body
    p
      :my-own-filter(addStart addEnd)
        Filter
        Body
