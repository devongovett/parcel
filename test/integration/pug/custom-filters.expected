<!DOCTYPE html>
<html lang="en">
  <body>
    <p>Start
Filter
Body
End
    </p>
  </body>
</html>